From 1e42ea0e629d6a013e6e7ad7f5d1bbfd2ea11338 Mon Sep 17 00:00:00 2001
From: Scott Moreau <oreaus@gmail.com>
Date: Wed, 9 Dec 2015 12:51:12 -0700
Subject: [PATCH] Avoid using templates in C code

libxslt uses templates and we cannot use templates in C code. Move the headers
using templates outside extern "C" clause.
---
 src/compiz.cpp | 6 +++---
 1 file changed, 3 insertions(+), 3 deletions(-)

diff --git a/src/compiz.cpp b/src/compiz.cpp
index 43da02f..e5aba0f 100644
--- a/src/compiz.cpp
+++ b/src/compiz.cpp
@@ -23,6 +23,9 @@
 #include <google/protobuf/io/zero_copy_stream_impl.h>
 #endif
 
+#include <libxslt/transform.h>
+#include <libxslt/xsltutils.h>
+
 extern "C"
 {
 #ifdef HAVE_CONFIG_H
@@ -36,9 +39,6 @@ extern "C"
 #include <sys/stat.h>
 #include <errno.h>
 
-#include <libxslt/transform.h>
-#include <libxslt/xsltutils.h>
-
 #include <locale.h>
 
 #include <compiz-core.h>
